frontend/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I can't comment on it directly (because the silly GitHub UI doesn't allow it) but if you look above, it looks like you're also committing an empty `bowtie/templates/migrate-report.html` file as part of this PR.